### DISCUSSION TITLE
With the release of .Net 9, will the current shiny version work with it?

### DISCUSSION THREAD

**Discussion by jmichas** [shinyorg/shiny#1534] [Q&A] [answered]
Just wondering if anyone has used shiny with .Net 9 yet and if it works or breaks.

**Comment by munkii** [comment]
I upgraded today and was able to discover, connect and read from the BT devices using existing code. That was however a debug build only but I'll assume Releasee will be fine too

**Comment by aritchie** [comment, marked as answer]
NET8 will run fine in NET9 in general.  It isn't a Shiny thing that you have to worry about.  The only thing that may come up is misalignment of versions with AndroidX stuff which I haven't seen yet.

**Comment by jmichas** [comment]
Ok, thanks for the info!
